src/components/secondPassphrase/secondPassphrase.css

```diff
@@ -0,0 +1,3 @@
+.hidden {
+  display: none;
+}
```

Comment:
since as I mentioned above, I'll have to use it, then this comment also needs to saty